packages/terra-form-input/tests/jest/InputField.test.jsx

```diff
@@ -2,145 +2,159 @@ import React from 'react';
 import IconHelp from 'terra-icon/lib/icon/IconHelp';
 import InputField from '../../src/InputField';
 
-it('should render a default InputField component', () => {
-  const textarea = <InputField inputId="test-input" label="Label" />;
-  const wrapper = shallow(textarea);
-  expect(wrapper).toMatchSnapshot();
-});
+describe('InputField', () => {
```

Comment:
This file looks like it had a lot of changes, but the largest change was wrapping the it statements within this describe block. These tests were flat/unscoped previously.